index.js

```diff
@@ -7,7 +7,7 @@ module.exports = bundler => {
   let publicURL = "./"
 
   const processBundle = (bundle) => {
-    let bundledFilename = path.join(publicURL, path.basename(bundle.name))
+    let bundledFilename = urlJoin(publicURL, path.basename(bundle.name))

```

Comment:
I've tested `url.resolve` & it solves this

Reply:
yeah, I considered `url.resolve` but the reason I didn't go for it was because
```javascript
url.resolve('/one/two/three', 'four') === '/one/two/four'
// whereas
urlJoin('/one/two/three', 'four') === '/one/two/three/four'
```
eg: https://repl.it/repls/MiserlySimilarDll

seemed that could cause confusion given in the main bundle parcel handles the above in the same way urlJoin does. What do you think?

Reply:
yeah, I've seen it too https://nodejs.org/api/url.html#url_url_resolve_from_to 🙄 but I'm unable to digest a dependency 😅

I'd like to know your opinion on solving it like this 🤔 
```
publicURL = publicURL.endsWith("/") ? publicURL : `{publicURL}/`
let bundledFilename = url.resolve(publicURL, path.basename(bundle.name))
```
eg: https://repl.it/repls/EagerQueasyControlpanel

Reply:
ok, will do it like that